content/developer/reference/cli.rst

```diff
@@ -425,7 +425,21 @@ HTTP
 .. option:: --proxy-mode
 
     enables the use of ``X-Forwarded-*`` headers through `Werkzeug's proxy
-    support`_. Only enable this when running behind a trusted web proxy!
+    support`_.
+
+    It ignores all ``X-Forwarded-*`` headers in case ``X-Forwarded-Host`` is
+    missing from the request.
```

Comment:
I think this language should be stronger. Something like "A properly configured proxy upstream must provide `X-Forwarded-Host` or --proxy-mode is ignored.

Reply:
Ah shit I only see your message now and the PR as been merged. I don't think using a stronger language is actually needed, the piece of information was missing and people had some expectations, now that the info is there I think they'll configure their proxies accordingly.

Reply:
We'll see, in case we continue to get messages on the matter we can still open another PR and change the wording :)